configuration/config-txt/memory.md

```diff
@@ -4,7 +4,9 @@
 
 GPU memory in megabytes, sets the memory split between the CPU and GPU; the CPU gets the remaining memory. The minimum value is `16`; the maximum value is `192`, `448`, or `944`, depending on whether you are using a 256MB, 512MB, or 1024MB Pi. The default value is `64`. For the Raspberry Pi 4, which is available in 1GB, 2GB and 4GB versions, the minimum and maximum value are the same as for a 1GB device.
 
-Setting `gpu_mem` to low values may automatically disable certain firmware features, as there are some things the GPU cannot do if it has access to too little memory. So if a feature you are trying to use isn't working, try setting a larger GPU memory split.
+gpu_memory refers to memory that must addressable to the GPU, which includes the VPU, HVS, Legacy codecs (e.g. H264), and camera, and on devices before the Raspberry Pi 4, the 3D system. The Raspberry Pi 4 3D system now has it's own Memory Mangement Unit (MMU) so textures and other GL resources are not allocated from the gpu_memory but Linux system memory instead. This means that gpu_memory can be set to a lower value, so even if you are using the H264 and camera then 128MB will probably be enough. On earlier models without the 3D MMU, you may need up to 256 or 512 in some more unusual cases. 
```

Comment:
"memory that must addressable to the GPU" - is there a word or something missing here?

Should there be an "MB" after `256` and `512` ?

Seems to be a mix-up between `gpu_mem` and `gpu_memory`?

Reply:
Should be fixed up.